component/rules.jsonnet

```diff
@@ -7,6 +7,15 @@ local defaultAnnotations = {
   syn_component: inv.parameters._instance,
 };
 
+local patch_node_filesystem_rules(rule) =
+  rule {
+    expr:
+      if std.startsWith(rule.alert, 'NodeFilesystem') then
```

Comment:
TODO: maybe switch to have a static list of alerts to patch. Right now all alerts which start with `NodeFilesystem` need the patch, but that might change in the future.